daemon/src/main/java/org/sonarlint/daemon/Daemon.java

```diff
@@ -100,27 +72,27 @@ public void stop() {
 
   public void start(int port) {
     try {
-      logger.info("Starting server on port " + port);
+      LOGGER.info("Starting server on port {}", port);
       ServerInterceptor interceptor = new ExceptionInterceptor();
 
       server = ServerBuilder.forPort(port)
-        .addService(ServerInterceptors.intercept(new ConnectedSonarLintImpl(logger), interceptor))
-        .addService(ServerInterceptors.intercept(new StandaloneSonarLintImpl(Utils.getAnalyzers(), logger), interceptor))
+        .addService(ServerInterceptors.intercept(new ConnectedSonarLintImpl(), interceptor))
+        .addService(ServerInterceptors.intercept(new StandaloneSonarLintImpl(Utils.getAnalyzers()), interceptor))
         .build()
         .start();
-      logger.info("Server started, listening on " + port);
+      LOGGER.info("Server started, listening on " + port);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2629)